libmcount/dynamic.c

```diff
@@ -454,22 +457,21 @@ static int do_dynamic_update(struct symtabs *symtabs, char *patch_funcs,
 	return 0;
 }
 
-static void finish_dynamic_update(struct mcount_disasm_engine *disasm)
+static void finish_dynamic_update()
```

Comment:
It's better to add explicit `void` for the function parameter.

Reply:
I missed it. thanks.